src/Ball.java

```diff
@@ -0,0 +1,17 @@
+public class Ball {
+
+    private int number;
```

Comment:
```suggestion
    private final int number;
```

Reply:
Al die finals zijn een goed idee. IntelliJ zei het ook al steeds, maar ik dacht vaak nee joh. Ik had final in mijn hoofd zitten als "na start programma zijn die niet meer te wijzigen" maar het is "na eerste initialisation kun je niet meer wijzigen" nogal een verschil.
Thanks!

Ik ga dit toepassen vanaf nu.